src/Congo.RazorPages/Extensions/ServicesExtensions.cs

```diff
@@ -13,9 +13,9 @@ public static void AddAppServices(this IServiceCollection services)
 
         public static void AddHttpClients(this IServiceCollection services)
         {
-            services.AddHttpClient("Congo", config =>
+            services.AddHttpClient(nameof(Congo), config =>
             {
-                config.BaseAddress = new Uri("/api");
+                config.BaseAddress = new Uri("https://localhost:44388/");
```

Comment:
```suggestion
                config.BaseAddress = new Uri("https://localhost:44388/api");
```

/api will always exist in the path